assets/controller.yaml

```diff
@@ -17,6 +17,8 @@ spec:
       maxSurge: 0
   template:
     metadata:
+      annotation:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
       labels:
```

Comment:
Can you add a comment here for the reader who isn't familiar with the enhancement but still can understand what it does?

Reply:
Sure thing, updated.